metadatastore/test/test_sample_data.py

```diff
@@ -1,42 +1,39 @@
 from metadatastore.examples.sample_data.common import apply_deadband, noisy
 from metadatastore.examples.sample_data import (temperature_ramp,
                                                 multisource_event)
-from metadatastore.utils.testing import mds_setup, mds_teardown
+from .utils import mds_setup, mds_teardown
 import numpy as np
 import itertools
-from nose.tools import raises
+import pytest
 
-def setup():
+
+def setup_function(function):
```

Comment:
it was taking seconds _per test_ with databroker, but if this is going fast enough, then it is fine.